.github/ISSUE_TEMPLATE/11-tag-lead-onboarding.yml

```diff
@@ -0,0 +1,63 @@
+name: TAG ENV Member Onboarding
```

Comment:
This is great, thanks for adding @leonardpahlke ! The only thing I can think of here is the member term usage - we had previously a discussion around that we don't have members in the TAG, only contributors plus TAG ENV Member/Contributor may be interpreted as it applies to anyone, not just the lead roles. Would a better term be TAG ENV Lead Role Onboarding or something similar?

Reply:
> Would a better term be TAG ENV Lead Role Onboarding

👍

Reply:
updated https://github.com/cncf/tag-env-sustainability/pull/552/commits/5a5dd9e7ba1ef0ef8b4d12685b0e1f889613f574